cmd/blockchaincmd/describe.go

```diff
@@ -276,6 +277,12 @@ func printAllocations(sc models.Sidecar, genesis core.Genesis) error {
 		t.SetTitle("Initial Token Allocation")
 		t.AppendHeader(table.Row{"Description", "Address and Private Key", "Amount (10^18)", "Amount (wei)"})
 		for address := range genesis.Alloc {
```

Comment:
nit: use `address, allocation := range genesis.Alloc` in this case following `genesis.Alloc[address]` can be replaced with `allocation` in the code